test/unit/ElectronicResources.test.js

```diff
@@ -25,7 +26,39 @@ describe('ElectronicResources', () => {
     expect(component.html()).to.be.null
   })
   it('should render nothing if electronic resources is empty array', () => {
-    const component = mount(<ElectronicResources electronicResources={[]}/>)
+    const component = mount(<ElectronicResources electronicResources={[]} />)
     expect(component.html()).to.be.null
   })
+  describe('show more/less button', () => {
+    it('should render with more than 3 electronic resources', () => {
+      const component = mount(<ElectronicResources electronicResources={fourResource} />)
+      expect(component.html()).to.include('See all')
+    })
+    it('should not render with less than 3 resources', () => {
+      const component = mount(<ElectronicResources electronicResources={oneResource} />)
+      expect(component.html()).to.not.include('See all')
+    })
+    it('should show all resources', () => {
+      const component = mount(<ElectronicResources electronicResources={fourResource} isTestMode />)
```

Comment:
Add a comment above this explaining why `isTestMode` is being passed here. It's in the component file itself, but it'd be nice to have it here as well so you don't flip back and forth between the two files.